services/worker/src/worker/job_runners/split/descriptive_statistics.py

```diff
@@ -495,7 +499,7 @@ def _compute_statistics(
         # TODO: count n_unique only on the first parquet file?
         nan_count, nan_proportion = nan_count_proportion(data, column_name, n_samples)
         n_unique = data[column_name].n_unique()
-        if n_unique <= MAX_NUM_STRING_LABELS:  # TODO: make relative
+        if n_unique / n_samples < MAX_PROPORTION_STRING_LABELS:
```

Comment:
nit: let's keep `<=` instead of `<`?

Reply:
did this https://github.com/huggingface/datasets-server/pull/2553/commits/e7dc6f44cdd33027d94a0b03e60308d50f3aad3c
and also switched 0.3 to 0.15, i feel like it's more realistic